src/Root.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { PeoplePage } from './pages/PeoplePage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="/home" element={<Navigate to="../" replace />} />
```

Comment:
The path '/home' is redirected to '../', which is the parent route. This might cause confusion if the intention is to redirect to the home page. Consider using '/' instead of '../' to redirect to the root path.